java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/common/DbzConnectorConfig.java

```diff
@@ -146,21 +146,21 @@ public DbzConnectorConfig(
                 // If cdc backfill enabled, the source only emit incremental changes, so we must
                 // rewind to the given offset and continue binlog reading from there
                 if (null != startOffset && !startOffset.isBlank()) {
-                    mysqlProps.setProperty("snapshot.mode", "schema_only_recovery");
+                    mysqlProps.setProperty("snapshot.mode", "recovery");
                     mysqlProps.setProperty(
                             ConfigurableOffsetBackingStore.OFFSET_STATE_VALUE, startOffset);
                 } else {
                     // read upstream table schemas and emit incremental changes only
-                    mysqlProps.setProperty("snapshot.mode", "schema_only");
+                    mysqlProps.setProperty("snapshot.mode", "never");
```

Comment:
From the [doc](https://debezium.io/documentation/reference/2.6/connectors/mysql.html),  `schema_only` should be replaced by `no_data`.

Reply:
updated to no_data